easybuild/easyconfigs/c/CMake/CMake-3.6.2-GCCcore-6.3.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.6.2'
```

Comment:
Not for gromacs no. I'm about to PR gromacs 2016.3 anyway... just installing/testing now